src/graph/GraphViewServer.ts

```diff
@@ -326,7 +329,7 @@ export class GraphViewServer extends EventEmitter {
           if (err.code === "ENOTFOUND") {
             // Not a valid endpoint
           } else {
-            firstValidError = firstValidError || err;
+            firstValidError = firstValidError || err || Object
```

Comment:
What's the purpose of `|| Object` here? Also - missing a semicolon at the end of the line

Reply:
Thanks for catching that.